src/test/java/serenityswag/inventory/InventoryPage.java

```diff
@@ -6,4 +6,8 @@ public class InventoryPage extends PageObject {
     public String getHeading() {
         return $(".title").getText();
     }
+
+    public String getShoppingCartBadge() {
```

Comment:
```suggestion
    public String numberShownOnCart() {
```
i find the name not great, you are returning text, not a shoppingCartBadge